client/src/components/IndivReview.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { FiMessageSquare } from 'react-icons/fi';
+import Avatar from 'react-avatar';
+import StarRatingComponent from 'react-star-rating-component';
+import PropTypes from 'prop-types';
+import styles from './style/indivReview.css';
+
+
+const IndivReview = ({ review }) => {
+  const {
+    review_food_rating, review_service_rating, review_ambience_rating,
```

Comment:
This would be easier to read if each thing was on it's own line.

Reply:
ok - I fixed it on all the files that had multiple properties being destructured